frontend/src/components/AuthForm/SignForm/SignUpForm.tsx

```diff
@@ -13,17 +13,20 @@ const schema = yup.object().shape({
     companyName: yup
         .string()
         .required("Company name required")
-        .min(6, "Company name too short!")
-        .max(20, "Company name too long!"),
+        .min(3, "Company name too short!")
+        .max(40, "Company name too long!")
+        .matches(/^\w[A-Za-zА-Яа-я\d!#$%&'*+\-/=?^_`]+$/, "Company name must be valid"),
```

Comment:
Mismatches validation rule from https://github.com/BinaryStudioAcademy/bsa-2020-feedgrabber/pull/145. Please, just copy-paste regex from that PR